backend/kesaseteli/applications/migrations/0015_add_youth_application_field_validation.py

```diff
@@ -0,0 +1,173 @@
+# Generated by Django 3.2.4 on 2021-12-14 10:53
+
+import applications.models
+import common.utils
+import django.core.validators
+from django.db import migrations, models
+import encrypted_fields.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("applications", "0014_set_default_schools"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="historicalyouthapplication",
+            name="email",
+            field=models.EmailField(max_length=254, verbose_name="email"),
+        ),
+        migrations.AlterField(
+            model_name="historicalyouthapplication",
+            name="encrypted_social_security_number",
+            field=encrypted_fields.fields.EncryptedCharField(
+                max_length=11, verbose_name="social security number"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="historicalyouthapplication",
+            name="first_name",
+            field=models.CharField(
+                max_length=128,
+                validators=[
+                    django.core.validators.RegexValidator(
+                        "^[A-Za-z0-9_',.ÄÅÖäåö-][^\\d!#$%&()*+/:;<=>?@[\\\\\\]_{|}~¡¿÷ˆ]+$"
+                    )
+                ],
+                verbose_name="first name",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="historicalyouthapplication",
+            name="is_unlisted_school",
+            field=models.BooleanField(),
+        ),
+        migrations.AlterField(
+            model_name="historicalyouthapplication",
+            name="last_name",
+            field=models.CharField(
+                max_length=128,
+                validators=[
+                    django.core.validators.RegexValidator(
+                        "^[A-Za-z0-9_',.ÄÅÖäåö-][^\\d!#$%&()*+/:;<=>?@[\\\\\\]_{|}~¡¿÷ˆ]+$"
```

Comment:
The migration file is autogenerated by Django. I don't think it'd be a good idea to use the NAMES_REGEX here because then we could retrospectively change old migration files if its value changed later. In my understanding old migration files should never change.

Reply:
true, I didnt know that